main.go

```diff
@@ -15,158 +15,115 @@ import (
 	"github.com/AdguardTeam/dnsproxy/upstream"
 	"github.com/AdguardTeam/golibs/log"
 	"github.com/ameshkov/dnscrypt/v2"
+
+	"github.com/BurntSushi/toml"
 	goFlags "github.com/jessevdk/go-flags"
 	"gopkg.in/yaml.v3"
 )
 
 // Options represents console arguments
+
+type ConsoleArgs struct {
+	// Print DNSProxy version (just for the help)
+	Version  bool   `long:"version" description:"Prints the program version"`
+	TomlPath string `short:"c" description:"Path to the toml configuration file (optional)." default:"/etc/dnsproxy/dnsproxy.toml" ` //
```

Comment:
Config file definitely should be a different pull request, it's quite hard to review a pull request when there're two different serious changes in a single one.

Reply:
Generally, I'd prefer `yaml` config over `toml` since we use `yaml` in other products already.

Reply:
Relevant feature request: https://github.com/AdguardTeam/dnsproxy/issues/57

We can discuss it there.